$NetBSD: patch-src_Makefile.am,v 1.1 2017/03/30 08:52:54 bouyer Exp $

Make the paths in gegrep & gfgrep absolute so if they are used without
${PREFIX}/${PKGGNUDIR}bin being in the path, they still work.

--- src/Makefile.am.orig	2016-11-21 18:31:31.000000000 +0100
+++ src/Makefile.am	2017-03-30 10:32:14.000000000 +0200
@@ -60,7 +60,7 @@
 	fi &&								\
 	sed -e 's|[@]SHELL@|$(SHELL)|g'					\
 	    -e "$$edit_substring"					\
-	    -e "s|[@]grep@|$$grep|g"					\
+	    -e "s|[@]grep@|$(PREFIX)/$(PKGGNUDIR)bin/$$grep|g"		\
 	    -e "s|[@]option@|$$option|g" <$(srcdir)/egrep.sh >$@-t
 	$(AM_V_at)chmod +x $@-t
 	$(AM_V_at)mv $@-t $@
